packages/experience-builder-sdk/src/utils.ts

```diff
@@ -112,3 +115,23 @@ export const getInsertionData = ({
     };
   }
 };
+
+export const generateRandomId = (count: number): string => {
+  const LETTERS = 'abcdefghijklmnopqvwxyzABCDEFGHIJKLMNOPQVWXYZ';
+  const NUMS = '0123456789';
+  const ALNUM = NUMS + LETTERS;
+
+  return times(count, () => ALNUM[random(0, ALNUM.length - 1)]).join('');
+};
+
+export const checkIfDesignComponent = ({
```

Comment:
Reads to me like a typo when thinking of other helper methods like `checkIsExperience` or was this intentional?
```suggestion
export const checkIsDesignComponent = ({
```

Reply:
It's intentional actual. It reads to me "check IF this component is a design component" hence `checkIfDesignComponent`